### ISSUE TITLE
1.18.2 version requires 1.19.x

### ISSUE THREAD

**Issue by Lancelotbronner** [BlossomMods/BlossomHomes#5]
See logged error, the `2.0.3+1.18.2` version is incompatible with Minecraft 1.18.2

```log
[19:57:58] [main/INFO]: Fix: add [], remove [], replace [[blossom-homes 2.0.3+1.18.2] -> add:blossom-homes 1 ([(-∞,∞)])]
[19:57:58] [main/ERROR]: Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'BlossomHomes' (blossom-homes) 2.0.3+1.18.2 with any version that is compatible with:
		 - minecraft 1.18.2
Unmet dependency listing:
	 - Mod 'BlossomHomes' (blossom-homes) 2.0.3+1.18.2 requires any 1.19.x version of 'Minecraft' (minecraft), but only the wrong version is present: 1.18.2!
```

**Comment by Lancelotbronner** [comment]
Probably (I hope) a metadata issue, this is preventing me from updating from FabricHomes

**Comment by CodedSakura** [comment]
it is just a metadata, you can force it to work using a [dependency override](https://fabricmc.net/wiki/tutorial:dependency_overrides)

though i'll probably have it fixed by this evening

**Comment by CodedSakura** [comment]
fixed in 4adcb40c14ef23af7b297e06327cbe76b60b9de7
